src/components/Layout/HeaderAndFooterLayout/HeaderAndFooterLayout.js

```diff
@@ -0,0 +1,6 @@
+import classes from "./HeaderAndFooterLayout.module.scss";
+const HeaderAndFooterLayout = ({ props: { children } }) => {
```

Comment:
You are destructuring props from props? And then children from second props? 
Can't see that you are using this component inside the project.